src/main/java/io/airbyte/testingtool/scenario/action/connection/ActionCreateConnection.java

```diff
@@ -34,6 +51,43 @@ public String getActionName() {
   @Override
   public void doActionInternal() throws ApiException {
     createConnection();
+    if (useNormalization) {
+      updateConnectionWithNormalization(connectionInstance.getConnectionId());
+    }
+  }
+
+  public void updateConnectionWithNormalization(UUID connectionId) throws ApiException {
+    var airbyteApiInstance = getAirbyteApiInstance();
+    ConnectionIdRequestBody connectionIdRequestBody = new ConnectionIdRequestBody();
+    connectionIdRequestBody.setConnectionId(connectionId);
+    ConnectionRead connectionRead = airbyteApiInstance.getAirbyteApi().getConnectionApi().getConnection(connectionIdRequestBody);
+
+    var destinationDefinitionId = destinationInstance.getDestinationDefinitionIdRequestBody()
+            .getDestinationDefinitionId();
+    DestinationDefinitionIdWithWorkspaceId definitionIdWithWorkspaceId = new DestinationDefinitionIdWithWorkspaceId();
+    definitionIdWithWorkspaceId.setDestinationDefinitionId(destinationDefinitionId);
+    definitionIdWithWorkspaceId.setWorkspaceId(airbyteApiInstance.getWorkspaceId());
+    var destinationDefinitionSpecification = airbyteApiInstance.getAirbyteApi()
+            .getDestinationDefinitionSpecificationApi()
+            .getDestinationDefinitionSpecification(definitionIdWithWorkspaceId);
+
+    if (Boolean.TRUE.equals(destinationDefinitionSpecification.getSupportsNormalization())) {
```

Comment:
Please create a method to `isSupportNormalization` and place it inside `DestinationInstance`

Reply:
done, thanks for the comment